terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -23,3 +23,16 @@ class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    slope = models.IntegerField(blank=True)
+    texture = models.CharField(blank=True)
+    color_hue_float = models.FloatField(blank=True)
+    color_hue_char = models.CharField(blank=True)
+    color_value = models.FloatField(blank=True)
+    color_chroma = models.IntegerField(blank=True)
+    conductivity = models.FloatField(blank=True)
+    structure = models.CharField(blank=True)
+    ph = models.FloatField(blank=True)
+    soc = models.FloatField(blank=True)
+    som = models.FloatField(blank=True)
+    sar = models.FloatField(blank=True)
+    bedrock = models.PositiveIntegerField(blank=True)
```

Comment:
changes needed for the remaining fields are i think similar as in previous comments so i'll leave them to you!